packages/react-app/src/components/common/UpdateSettings.jsx

```diff
@@ -129,6 +131,13 @@ export const UpdateSettings = ({ close }) => {
                   isChecked={neverShowClaims}
                   onChange={e => setNeverShowClaims(e.target.checked)}
                 />
+                <Text mb={2}>Fetch token list with balance</Text>
```

Comment:
Can we change this to `Disable balances in token list` ?
And let it be unchecked by default, i.e., balances are fetched by default.

Reply:
Sure thing.

Reply:
Did you measure the latency users experience due to getting the balances for such amount of tokens? If it is significant is it worth to keep the configuration to disable getting the balances as default?